.vscode/tasks.json

```diff
@@ -0,0 +1,24 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
```

Comment:
Let's remove this `.vscode` folder from the repo, since we didn't have one in here already.

Reply:
Will do.